src/test/java/study/step4/RandomMoveStrategyTest.java

```diff
@@ -0,0 +1,24 @@
+package study.step4;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import study.step3.model.strategy.MoveStrategy;
+
+public class RandomMoveStrategyTest {
```

Comment:
```suggestion
public class MoveStrategyTest {
```
MoveStrategy에 대한 테스트라고 해야 맞지 않을까요?